state/session/reducer.js

```diff
@@ -1,26 +1,24 @@
+import cookie from 'cookie_js';
 import * as types from './types';
 
+const checkSessionIdInCookie = cookie.get('session_id');
```

Comment:
I think this variable should be named as `sessionId` because it's just session id value, not a function which returns session id

Reply:
I agree with you. Fixed